components/match2/wikis/heroes/match_summary.lua

```diff
@@ -292,8 +292,11 @@ function CustomMatchSummary.createBody(match)
 	if not Table.isEmpty(championBanData) then
 		local championBan = ChampionBan({isBan = true})
 
-		for gameIndex, banData in ipairs(championBanData) do
-			championBan:banRow(banData, gameIndex, banData.numberOfBans, match.date)
+		for gameIndex in ipairs(match.games) do
+			local banData = championBanData[gameIndex]
+			if banData then
```

Comment:
why not let banrow function just return in case of empty bandata?

mind with a few additional changes to banrow we could (double) curry here